e2e-tests/portal/cypress/integration/journeys/checker/cover-end-date/dealFacilityNotAcknowledged.js

```diff
@@ -0,0 +1,121 @@
+const dealAllAcknowledged = require('./deal');
```

Comment:
same as above, can we inherit anything else from other files?

Reply:
removed